code401Challenges/src/main/java/code401Challenges/tree/Tree.java

```diff
@@ -100,4 +99,26 @@ public Tree(Node<T> root) {
         return valuesArr;
     }
 
+    // ======= Breadth First Traversal, using a queue, for numbers
+    public void breadthFirstTraverse (Node<Integer> root) {
```

Comment:
You don't need to pass in a root node since you will be calling this method on a Tree, and all Trees have a root